dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/test-applications/TxFaultTolerance.war/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
```

Comment:
...not going to comment on every date...

Reply:
no. that would be silly.